README.md

```diff
@@ -120,6 +121,7 @@ Parameters: `withAuxInfo: boolean`. Optional, by default, `false`.
 
 `setMolecule(structure: string): Promise<void>` – draws passed structure on the canvas. Before drawing passed structure, current structure is removed.  
 Parameters: `structure: string`. Structure is a string in any supported format.
+Parameters: `rescale: boolean`. rescale is a boolean, default will be true. false means struct will not rescale on canvas
```

Comment:
It's not necessary to add one more `Parameters` word here, as we have it already, it has to be a parameters list:
```
Parameters: 
- 'structure: string' ...
- 'rescale: boolean' ...
```